src/views/About/index.jsx

```diff
@@ -15,6 +15,8 @@ const About = () => {
     versionService.getVersion().then(setVersion);
   }, []);
 
+  const getColor = (cssVar) => getComputedStyle(document.documentElement).getPropertyValue(cssVar);
```

Comment:
This is now unused in the file, please delete it.

Reply:
getColor was still being used, but it was unneeded, so I'll change it.